packages/react/src/components/Masthead/HeaderNavContainer.js

```diff
@@ -76,6 +126,9 @@ const HeaderNavContainer = ({ children }) => {
     if (io) {
       io.observe(contentLeftRef.current);
       io.observe(contentRightRef.current);
+      setMenuItems(
+        document.querySelector('.bx--header__menu-bar').querySelectorAll('li')
+      );
```

Comment:
Instead of saving menu items in a state, I think we can qSA for menu items from `contentRef`.

Reply:
thanks! just updated!